federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -78,6 +80,34 @@ func parseFederationReplicaSetReference(frs *extensionsv1.ReplicaSet) (*fed.Fede
 	return &frsPref, nil
 }
 
+func initReplicaSetProcessDelays(replicaSetProcessDelays string) {
```

Comment:
don't we have validation in place with proper error handling? `glog.Fatal`s are ugly, especially if they can be triggered by a component config (can they?).